www/index.html

```diff
@@ -87,9 +87,16 @@
             <span class="nav-header-tab nav-header-text" id="insert-tab">
               <i class="fa fa-desktop"></i> Insert
             </span>
-            <span class="nav-header-tab nav-header-text" id="other-tab">
-                <i class="fa fa-ellipsis-h"></i> Other
-              </span>
+            <div class="list">
```

Comment:
On Mac it doesn't really look nice.
![bug](https://user-images.githubusercontent.com/2982784/44311693-bfa29c00-a3a0-11e8-8950-9ea1f552bd65.gif)

It also doesn't behave the same as the video in the PR. takes 2 clicks to go to the selection when changing the selection.

@saintsoldierx should comment but my suggestion would be not use a select tag for the extra tags.